src/classes/Ingredient.js

```diff
@@ -1,11 +1,30 @@
 class Ingredient {
-  constructor(id, name, cost, amount, unit) {
-    this.id = id
-    this.name = name
-    this.estimatedCostInCents = cost
-    this.amount = amount
-    this.unit = unit
+  constructor(data) {
+    this.id = data.id
+    this.name = data.name
+    this.cost = data.cost
+    this.amount = data?.amount
+    this.unit = data?.unit
```

Comment:
Hey! I believe `optional chaining` is for nested object properties, and since `amount` and `unit` are just properties they will already default to `undefined`. I tried without it and console logged in the test the ingredient that didn't have `amount` and `unit` given. The result is `undefined`. Here is a screenshot below:

![image](https://user-images.githubusercontent.com/108706408/199750828-ca6a00fd-8255-431e-9fc4-5e45fdbd6416.png)

Reply:
Sounds good! I can push up those changes.